yarn.lock

```diff
@@ -367,10 +367,10 @@
   version "1.1.3"
   resolved "https://registry.yarnpkg.com/@nodelib/fs.stat/-/fs.stat-1.1.3.tgz#2b5a3ab3f918cca48a8c754c08168e3f03eba61b"
 
-"@okta/okta-auth-js@^3.1.0":
-  version "3.1.0"
-  resolved "https://registry.yarnpkg.com/@okta/okta-auth-js/-/okta-auth-js-3.1.0.tgz#1d66868bfdc34e2969acf42f32fa238ac5ac4d78"
-  integrity sha512-6o0SSD6iH3T3vhUhAJByoPbpbLAfZl7uFz6S6If0gP872O39mrrj4spmEMtX8ygQheBnt23KFa7YZpVwAjB/9g==
+"@okta/okta-auth-js@3.1.1":
```

Comment:
why is this change included ?

Reply:
seeing errors caused by https://github.com/okta/okta-signin-widget/pull/1166 during testcafe test. I'm not sure why it didn't happy in travis.
this commit will be removed after 4.0.2 backported.